src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -0,0 +1,150 @@
+          class="kuiLink kuiSubText"
+          ng-click="stepTimeField.fetchTimeFieldOptions()"
+          kbn-accessible-click
+          aria-describedby="timeFilterFieldSelectLabel"
+        >
+          Refresh
+        </a>
+
+        <p
+          ng-if="stepTimeField.isFetchingTimeFieldOptions"
+          class="kuiText kuiSubduedText"
+          aria-label="Please wait while we fetch your time field options"
+          aria-describedby="timeFilterFieldSelectLabel"
+        >
+          <span
+            aria-hidden="true"
+            class="kuiIcon fa-spinner fa-spin"
+          ></span>
+        </p>
+      </div>
+
+      <div class="kuiVerticalRhythmSmall kuiVerticalRhythm">
+        <select
+          id="timeFilterFieldSelect"
+          aria-describedby="timeFilterFieldSelectHelpText"
+          class="kuiSelect kuiSelect--large"
+          data-test-subj="createIndexPatternTimeFieldSelect"
+          ng-disabled="stepTimeField.isTimeFieldSelectDisabled()"
+          ng-required="stepTimeField.timeFieldOptions.length"
+          ng-options="option as option.display disable when option.isDisabled for option in stepTimeField.timeFieldOptions"
+          ng-model="stepTimeField.selectedTimeFieldOption"
+        ></select>
+      </div>
+
+      <p
+        id="timeFilterFieldSelectHelpText"
+        class="kuiText kuiSubduedText kuiVerticalRhythm"
+      >
+        The Time Filter will use this field to filter your data by time.
+      </p>
+    </div>
+
+    <!-- Advanced options -->
+    <toggle-panel
+      class="kuiVerticalRhythm"
+      ng-if="stepTimeField.canEnableExpandWildcard()"
+      is-collapsed="!stepTimeField.isAdvancedOptionsVisible"
+      on-toggle="stepTimeField.onToggleAdvancedOptions"
+      button-text="{{stepTimeField.getAdvancedOptionsButtonLabel()}}"
+      button-text="Advanced Time Field filter options"
+    >
+      <!-- Expand index pattern checkbox -->
+      <label
+        class="kuiLabel kuiVerticalRhythmSmall"
+        for="expandIndexPatternCheckBox"
+      >
+        Expand index pattern (DEPRECATED)
+      </label>
+
+      <label class="kuiCheckBoxLabel kuiVerticalRhythmSmall kuiVerticalRhythm">
+        <input
+          id="expandIndexPatternCheckBox"
+          aria-describedby="expandIndexPatternCheckBoxHelpText"
+          class="kuiCheckBox"
+          type="checkbox"
+          data-test-subj="createIndexPatternEnableExpand"
+          ng-model="stepTimeField.expandWildcard"
+        >
+        <span class="kuiCheckBoxLabel__text">
+          Expand index pattern when searching
+        </span>
+      </label>
+
+      <!-- Checkbox help text -->
+      <div
+        id="expandIndexPatternCheckBoxHelpText"
+        class="kuiVerticalRhythm"
+      >
+        <p class="kuiText kuiStatusText kuiStatusText--error kuiVerticalRhythmSmall">
+          Recent changes to Elasticsearch have made this option largerly unnecessary. It will likely be removed in a future version of Kibana.
```

Comment:
It's safe to remove "likely" from this line, we're trying to figure out the soonest we can remove it right now https://github.com/elastic/elasticsearch/issues/25577